audit/projects/kubernetes-public/secrets/k8s-infra-ci-robot-github-token/versions.json

```diff
@@ -1,11 +1,20 @@
 [
+  {
+    "createTime": "2021-08-03T15:53:57.204242Z",
+    "etag": "\"15c8a9b18a2d12\"",
+    "name": "projects/127754664067/secrets/k8s-infra-ci-robot-github-token/versions/2",
+    "replicationStatus": {
+      "automatic": {}
+    },
+    "state": "ENABLED"
+  },
   {
     "createTime": "2021-06-15T21:10:17.454396Z",
-    "etag": "\"15c4d466a06f3c\"",
+    "etag": "\"15c8a9b398bcc0\"",
     "name": "projects/127754664067/secrets/k8s-infra-ci-robot-github-token/versions/1",
     "replicationStatus": {
       "automatic": {}
     },
-    "state": "ENABLED"
+    "state": "DISABLED"
```

Comment:
This was me, I had to rotate k8s-infra-ci-robot's github token after realizing we had exposed it 😬 

https://github.com/kubernetes/k8s.io/pull/2448 removed the exposure